templates/tower-project.yaml

```diff
@@ -428,12 +421,33 @@ Resources:
               KMSMasterKeyID: !GetAtt EncryptionKeyStack.Outputs.KeyArn
       LifecycleConfiguration:
         Rules:
-          - Id: UserDefinedDataLifecycleRule
-            Status: !Ref EnableScratchLifeCycle
-            ExpirationInDays: !Ref ScratchLifecycleExpiration
+          - Id: LifecycleStorageClassTransition
             Transitions:
               - TransitionInDays: !Ref ScratchLifecycleTransition
                 StorageClass: !Ref ScratchLifecycleStorageClass
+          - Id: DataExpirationForWorkDirectory
+            Status: !Ref EnableScratchDataExpiration
+            ExpirationInDays: 30
+            Prefix: work/
+          - Id: DataExpirationIn07Days
+            Status: !Ref EnableScratchDataExpiration
+            ExpirationInDays: 7
+            Prefix: 07_days/
+          - Id: DataExpirationIn14Days
+            Status: !Ref EnableScratchDataExpiration
+            ExpirationInDays: 14
+            Prefix: 14_days/
+          - Id: DataExpirationIn30Days
+            Status: !Ref EnableScratchDataExpiration
+            ExpirationInDays: 30
+            Prefix: 30_days/
+          - Id: DataExpirationIn60Days
+            Status: !Ref EnableScratchDataExpiration
+            ExpirationInDays: 60
+            Prefix: 60_days/
+          - Id: DataExpirationIn90Days
+            Status: !Ref EnableScratchDataExpiration
+            ExpirationInDays: 90
```

Comment:
Just as a heads-up, this kind of repetition can be avoided if we switch to a Jinja template (see snippet below). I'm wondering if we should make the switch. 

```yaml
          {%- for num in sceptre_user_data.expire_days %}
          - Id: DataExpirationIn{{ num }}Days
            Status: !Ref EnableScratchDataExpiration
            ExpirationInDays: {{ num }}
            Prefix: {{ num }}_days/
          {%- endfor %}
```

Reply:
I think this could be a nice switch - I also see this as a good learning experience for adding jinja templates within these templates.  If there is time in the next 2 weeks prior to the merge, that'd be nice to tackle this, but this technically isn't part of the acceptance criteria.

Reply:
@BrunoGrandePhD in the example above, where does the data `sceptre_user_data.expire_days` come from?

Reply:
@BWMac I created a ticket here: https://sagebionetworks.jira.com/browse/IBCDPE-488